### PR TITLE
change istio service path prefix /workload to /servicemesh

### PR DESCRIPTION
**Pull request by LERSONG**

### CODE REVIEW

File: client/api/endpoints/istio-destination-rule.api.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { autobind } from "../../utils";
 import { KubeObject } from "../kube-object";
 import { KubeApi } from "../kube-api";
 import { number, boolean } from "yargs";
+import {apiIstio} from "..";
```
Comment (withlin) [inline client/api/endpoints/istio-destination-rule.api.ts:5]: Can you explain what is imported?

File: client/kube-object.store.ts [hunk 2]
```diff
@@ -170,6 +171,15 @@ export abstract class KubeObjectStore<
   }
 
   async remove(item: T) {
+    if (item.selfLink.startsWith("/apis/networking.istio.io/v1beta1")) {
+      const itemApi = apiManager.getApi(item.selfLink)
+      await itemApi.delete({ name: item.getName(), namespace: item.getNs() })
+          .then((res) => {
+             Notifications.ok(item.kind + " " + item.getName() +" delete succeeded")
```
Comment (withlin) [inline client/kube-object.store.ts:178]: Must be success delete here?

**Review by withlin — CHANGES_REQUESTED** [review]
Some minor nits for the pr

File: client/kube-object.store.ts (hunk 2 above)
Comment (LERSONG) [reply client/kube-object.store.ts:178]: done

File: client/api/endpoints/istio-destination-rule.api.ts (hunk 1 above)
Comment (LERSONG) [reply client/api/endpoints/istio-destination-rule.api.ts]: done

**Review by withlin — APPROVED** [review]
/lgtm

**Pull request merged**